sdk/python/resources/workspace/workspace-managed-network.ipynb

```diff
@@ -0,0 +1,371 @@
+    "\n",
+    "Note: We can optionally add outbound rules when configuring the workspace (examples of rules shown below)\n",
+    "Note: PrivateEndpointOutboundRule is the only supported type of outbound rule for AIO isolation mode"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "managed_network = ManagedNetwork(\n",
+    "    isolation_mode=IsolationMode.ALLOW_INTERNET_OUTBOUND, outbound_rules=[]\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.4 Create a basic workspace using the managed network\n",
+    "To create the workspace, we will define the following attributes\n",
+    "- `name` - Name of the workspace\n",
+    "- `managed_network` - The managed network settings configured for the workspace (from cell above)\n",
+    "- `location` - The Azure [location](https://azure.microsoft.com/en-us/global-infrastructure/services/?products=machine-learning-service) for workspace. For e.g. eastus, westus etc.\n",
+    "\n",
+    "Using the `MLClient` created earlier, we will create the workspace. This command will start workspace creation and provide a confirmation."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "name": "basic_workspace_name"
+   },
+   "outputs": [],
+   "source": [
+    "# Creating a unique workspace name with current datetime to avoid conflicts\n",
+    "\n",
+    "mvnet_workspace_name = \"mlw-mvnet-prod-\" + datetime.datetime.now().strftime(\n",
+    "    \"%Y%m%d%H%M\"\n",
+    ")\n",
+    "\n",
+    "ws_mvnet = Workspace(\n",
+    "    name=mvnet_workspace_name,\n",
+    "    location=\"eastus\",\n",
+    "    managed_network=managed_network,\n",
+    ")\n",
+    "\n",
+    "ws_mvnet = ml_client.workspaces.begin_create(ws_mvnet).result()\n",
+    "print(ws_mvnet)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1.5 Update the isolation mode to AllowOnlyApprovedOutbound (AOAO) \n",
+    "The isolation mode for the workspace managed network can be updated using workspace update to be the more restrictive AOAO mode. For a workspace with AOAO mode enabled, internet outbound traffic is restricted from managed Vnet, unless specified by to an approved list of services, service tags, or FQDNs. After this update in mode, we can see the difference in outbound rules of the managed network. Some additional rules are added to the network settings as `Required` outbound rules since communication outbound is removed so PEs are necessitated to certain workspace-associated resources and certain service tags are required for some system communication. We can also see that these required rules are `Inactive` and the managed network is also in `Inactive` state as managed network creation is deferred until a compute is created or the network is provisioned.\n",
```

Comment:
```suggestion
    "The isolation mode for the workspace managed network can be changed to the more restrictive AOAO mode via workspace update. For a workspace with AOAO mode enabled, internet outbound traffic is blocked from the managed network, unless specified by an approved list of services, service tags, or FQDNs. After this update in isolation mode, we can see the difference in outbound rules of the managed network. Some additional rules are added to the network settings as `Required` outbound rules since communication outbound is removed so PEs are necessitated to certain workspace-associated resources and certain service tags are required for some system communication. We can also see that these required rules are `Inactive` and the managed network is also in `Inactive` state as managed network creation is deferred until a compute is created or the network is provisioned. Once the isolation mode is changed to AOAO, it can't not be reverted back to AIO or Disabled.\n",
```

Reply:
nice thought, thanks. applied it